reference/library/src/morpheus-master/sass/modules/_customization.scss

```diff
@@ -0,0 +1,20 @@
+#content {
+  #col1of2 {
+    width: 68%;
```

Comment:
Is it worth extracting these out in the main skin into variables so it's something configurable in the skin?

Then we can just put our variables into the _customisations.scss